dev-packages/e2e-tests/test-applications/solidstart/tests/performance.client.test.ts

```diff
@@ -0,0 +1,93 @@
+import { expect, test } from '@playwright/test';
+import { waitForTransaction } from '@sentry-internal/test-utils';
+
+test('sends a pageload transaction', async ({ page }) => {
+  const transactionPromise = waitForTransaction('solidstart', async transactionEvent => {
+    return !!transactionEvent?.transaction && transactionEvent.contexts?.trace?.op === 'pageload';
+  });
+
+  await page.goto('/');
+  const pageloadTransaction = await transactionPromise;
+
+  expect(pageloadTransaction).toMatchObject({
+    contexts: {
+      trace: {
+        op: 'pageload',
+        origin: 'auto.pageload.browser',
```

Comment:
Follow up task: We likely want to emit `auto.pageload.solidstart.router` here.

Reply:
Thanks, will update this in a follow up PR.